landoui/assets_src/css/lando.scss

```diff
@@ -24,6 +24,17 @@ body {
 
 main {
   flex: 1 0 auto;
+  padding: 1em 0;
+}
+
+h1 {
+  font-size: 2em;
+  font-weight: bold;
+  margin-bottom: .25em;
+}
+
+p {
```

Comment:
This style seems awfully generic. For example on the revisions page, the section headings are really tall for no reason.

# Before 
![before2_p_margin](https://user-images.githubusercontent.com/7828780/28834424-2e5e4a0a-76b0-11e7-8627-78c5d1dd5ea1.PNG)

# After
![after_p_margin](https://user-images.githubusercontent.com/7828780/28834434-33b902c4-76b0-11e7-9e2c-6f69468a2fca.PNG)